R/join_keys-parents.R

```diff
@@ -73,11 +73,15 @@ parents.teal_data <- function(x) {
 #' parents(jk)["ds6"] <- "ds5"
 #' parents(jk)["ds7"] <- "ds6"
 `parents<-.join_keys` <- function(x, value) {
-  checkmate::assert_list(value, types = "character", names = "named")
+  checkmate::assert_list(value, types = c("character"), names = "named")
```

Comment:
I think this extra call is unnecessary. What's special about adding `c()`?

```suggestion
  checkmate::assert_list(value, types = "character", names = "named")
```